stories/Line.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react'
+import { LinkProps } from '../src/types'
+
+const Line = (props: LinkProps) => {
+  const { id, label, source, target, hover, quadraticPoint } = props
+
+  return (
+    <g>
+      <line
+        id={`${id}`}
+        x1={source.x}
+        y1={source.y}
+        x2={target.x}
+        y2={target.y}
+        strokeWidth={hover ? 5 : 1}
+        stroke={source.color}
+      />
+      <foreignObject {...quadraticPoint} width={50} height={100}>
+        <div
+          style={{
+            textAlign: 'center',
+            color: source.color,
+            fontWeight: hover ? 'bold' : 'lighter',
+          }}
+        >
+          {label}
+        </div>
+      </foreignObject>
+    </g>
+  )
+}
+
+export default memo(
```

Comment:
You can remove the memo from stories : this is hard to read and add nothing to the user (if he needs to memo he will use the react documentation)